### PR TITLE
Updating footer

### LINKED ISSUE #436: Footer not up to date on Teams Page and Suggested Book Page
GSSOC 2021 PARTICIPANT

## **Describe the bug**
The footer on the teams and Suggested Books page is missing certain links under features.
- A clear and concise description of what the bug is.

## **Possible solution**
Update the footer of both these pages like in the home page.
- Describe the solution you thought of.

## **Screenshots**
Home Page

tempsnip

- If applicable, add screenshots to help explain your problem.

### PR DESCRIPTION
**Pull request by sohini3062**
# Description
Updated the footers of booklist page and teams page
Please include a summary of the change and which issue is fixed.
Fixes #436
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before : footer_Team Page Before
teampage_prev

    - After :footer_Team Page After 
teams_updated

Booklist Page's footer before
bookpage_previous
Booklist page's footer after
updated_booklist

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: booklist.html [hunk 1]
```diff
@@ -78,55 +79,67 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
...
-                <div class="col-lg-3 col-md-3 col-sm-6 col-12">
-                    <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-
-                    <a href="https://github.com/Algo-Phantoms"><i class="fa fa-github p-2 text-white" aria-hidden="true"
-                            style="background:#3caea3; border-radius: 50%;"></i>
-                    
 
+                <div class="col-lg-3 col-md-3 col-sm-6 col-12 socialicons">
+                    <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
+                    <a href="https://github.com/Algo-Phantoms" target="_blank"><i  class="fa fa-github p-2 socialicon " aria-hidden="true"
```
Comment (harikesh409) [inline booklist.html:123]: Remove the trailing space after socialicon.

File: team.html [hunk 2]
```diff
@@ -339,19 +339,21 @@ <h2>Nakul Sharma</h2>
...
     <footer id="footer">
         <div class="container">
             <div class="row subscribe-box py-5">
                 <div class="col-lg-4 col-md-4 col-sm-12 col-12 offset-lg-1">
                     <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
+
+                    <form action="mailto:algophantom0101@gmail.com">
                     <form action="#">
```
Comment (harikesh409) [inline team.html:352]: Remove this form action as you already added a new one.

File: team.html [hunk 3]
```diff
@@ -385,15 +387,27 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Useful Li
                         </li>
                     </ul>
                 </div>
+
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features</h5>
+
+                    <ul>
+                        <a class="text-white mb-lg-3" href="https://www.codechef.com/ide" target="_blank">Code Editor</a><br>
```
Comment (harikesh409) [inline team.html:395]: Already few links are present and you added again just check and remove the duplicates.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: team.html (hunk 2 above)
Comment (sohini3062) [reply team.html:350]: Removed this one.

File: team.html (hunk 3 above)
Comment (sohini3062) [reply team.html:393]: removed the duplicate links

**Review by harikesh409 — APPROVED** [review]

**Review by ravikr126 — APPROVED** [review]

**Pull request merged**